docs/user/alerting/alerting-getting-started.asciidoc

```diff
@@ -169,12 +169,15 @@ If you are using an *on-premises* Elastic Stack deployment with <<using-kibana-w
 
 To access alerting in a space, a user must have access to one of the following features:
 
+* Alerting
 * <<xpack-apm,*APM*>>
+* <<logs-app,*Logs*>>
 * <<metrics-app,*Metrics*>>
 * <<xpack-siem,*Security*>>
 * <<uptime-app,*Uptime*>>
 
-See <<kibana-feature-privileges, feature privileges>> for more information on configuring roles that provide access to these features. 
+See <<kibana-feature-privileges, feature privileges>> for more information on configuring roles that provide access to these features.
+Also note that a user will need privileges for the Actions & Connectors feature to attach actions to an alert.
```

Comment:
If "Actions & Connectors" refers what's in the screenshot below, then spell out "and" and use bold.

<img width="446" alt="Screen Shot 2020-11-02 at 3 18 20 PM" src="https://user-images.githubusercontent.com/33642766/97929785-c79b4c00-1d1e-11eb-9290-2b629673dc41.png">

Reply:
Done.